pkg/epp/handlers/streamingserver.go

```diff
@@ -322,6 +323,7 @@ func (s *StreamingServer) HandleRequestBody(
 	if !ok {
 		return reqCtx, errutil.Error{Code: errutil.BadRequest, Msg: "model not found in request"}
 	}
+	metrics.IncRunningRequests(model)
```

Comment:
if the request gets rejected by EPP (fast response), will we still get this call? @kfswain perhaps you know?